pkg/proxy/accesslog/record.go

```diff
@@ -68,15 +68,34 @@ const (
 	VersionIPV6
 )
 
-// EndpointInfo contains information about the endpoint sending/receiving the flow
+// EndpointInfo is used for the SourceEndpoint and DestinationEndpoint field
+// and contains information about the sending respectively receiving endpoint.
```

Comment:
Nit: The wording is a bit unclear to me
`sending respectively receiving endpoint.`